features/networking/service.feature

```diff
@@ -105,3 +105,27 @@ Feature: Service related networking scenarios
     And the output should not contain:
       | <%= cb.service_ip %> |
 
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23895
+  @admin
+  Scenario: User cannot access the MCS by creating a LoadBalancer service that points to the MCS
+    Given I select a random node's host
+    Given I have a project
+    And SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group
+    And I have a pod-for-ping in the project
+    And the pod named "hello-pod" becomes ready
+    #Creating laodbalancer service that points to MCS IP
+    When I run the :create_service_loadbalancer client command with: 
```

Comment:
here I saw you create loadbalance service is mapping hello-pod ip and it's not using the MCS IP, So I guess the ep is not expected. you can check the endpoints if is correct 'oc get ep'
IMO, in order to make the loadbalance service is mapping to MCS ip, there are two methods:
1. using 'oc path' to update the endpoint ip after created the service.   
2. Create 'Loadbalancer' service in `openshift-machine-config-operator` project , like 
    ```
     $oc create service loadbalancer mcs --tcp=23600:23623` 
     $ `oc edit svc mcs`  to update 'app: mcs' to 'k8s-app: machine-config-server' for `selector`
     ####check the ep###
     # oc get ep
     NAME   ENDPOINTS             AGE
      mcs    10.73.130.109:23623   8m
```

Reply:
Hmm, guess i have to recheck it and will get back to you. But option 2 wont be feasible to configure as a test user due to restrictions. If i use option 1st, i can try following to add in oc edit ep
subsets:
- addresses:
  - ip: 10.0.131.58

Reply:
@zhaozhanqi I implemented option 1 you suggested.Could you look at revised scenario? https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3-smoke/378/console